src/extension/appLauncher.ts

```diff
@@ -174,6 +177,8 @@ export class AppLauncher {
             mobilePlatformOptions.isDirect = true;
         }
 
+        this.launchScenariosManager = new LaunchScenariosManager(this.workspaceFolder.uri.fsPath);
```

Comment:
Can we do this in class constructor? I don't think `fsPath` will change when the application starts.